pages/search.tsx

```diff
@@ -28,18 +45,13 @@ const Search = () => {
             </form>
             <div>
                 <ul>
-                {coins.map((coin: Coin) => (
-                    <li key={coin.id}>{coin.id}</li>
+                {coins?.map((coin: Coin) => (
+                    <li key={coin.id}>{coin.id} - rank: {coin.market_cap_rank}</li>
                 ))}
                 </ul>
             </div>
```

Comment:
You could extract this out to a `SearchResults` component and pass the coin list in as props and conditionally render based on coin list.

Reply:
I'll do that when I setup adding coins to portfolio functionality.